plugins/outputs/dynatrace/README.md

```diff
@@ -16,7 +16,7 @@ The Dynatrace exporter may be enabled by adding an `[[outputs.dynatrace]]` secti
 All configurations are optional, but if a `url` other than the OneAgent metric ingestion endpoint is specified then an `api_token` is required.
 To see all available options, see [Configuration](#configuration) below.
 
-### Running alongside Dynatrace OneAgent
+### Running alongside Dynatrace OneAgent (preferred)
```

Comment:
This should be at the same level as "Running standalone" below, so please add another `#` down there, so both are under "Getting started".

Reply:
Sounds reasonable.